conclave-enclave/src/main/kotlin/com/r3/conclave/enclave/internal/StreamEnclaveHostInterface.kt

```diff
@@ -0,0 +1,189 @@
+package com.r3.conclave.enclave.internal
+
+import com.r3.conclave.common.internal.*
+import com.r3.conclave.mail.internal.writeInt
+import com.r3.conclave.utilities.internal.getAllBytes
+import java.io.InputStream
+import java.io.OutputStream
+import java.nio.ByteBuffer
+import java.util.concurrent.ArrayBlockingQueue
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.Executors
+
+class StreamEnclaveHostInterface(
+        private val outputStream: OutputStream,     // Messages going to the host
+        private val inputStream: InputStream,       // Messages arriving from the host
```

Comment:
rename these `toHost` and `fromHost` respectively. Likewise for the other class.